docs/tutorial/quickstart.md

```diff
@@ -12,16 +12,15 @@ Create a new Django project named `tutorial`, then start a new app called `quick
 
     # Create a virtualenv to isolate our package dependencies locally
     virtualenv env
-    source env/bin/activate  # On Windows use `env\Scripts\activate`
+    . env/bin/activate  # On Windows use `env\Scripts\activate`
 
     # Install Django and Django REST framework into the virtualenv
-    pip install django
-    pip install djangorestframework
+    pip install django djangorestframework
```

Comment:
I'd probably slightly prefer to keep these as two separate commands. (Easier to not notice a mistake with them together, and more obvious where the problem lies if one of them failed to install)